webpack/app/js/layout.js

```diff
@@ -478,7 +479,7 @@ export default class LayoutView {
       $('#slider').off("slide");
       $('#slider').on("slide", function(event, ui) {
         column.attr('data-column-size', ui.value);
-        le.trigger('modified.layout');
+        le_resize.trigger('modified.layout');
```

Comment:
please look at the comments about `le_delete`, the same apply here.

Reply:
"le_resize" is needed cause "le" is not reached into function

Reply:
yes, that's why you should use the `.bind(this)` code when add an event, look at the other comments please.